_posts/2016-03-01-Docker-meets-Continuous-Deployment.md

```diff
@@ -15,17 +15,17 @@ tags:
 - TeamCity
 --- 
 
-About one year ago I had my first contact with Docker. This new kid on the block promised to relieve our poor computers from installation of all tools, languages, dependencies and operating systems. Isolated run environments emerged on developers computers.
+About one year ago I had my first contact with Docker. This new kid on the block promised to relieve our poor computers from installation of all tools, languages, dependencies and operating systems. Isolated run environments emerged on developers' computers.
 
-A conservative approach of my ops teammates didn't change my joy of using Docker. Despite many people describing Docker as tool written by developers for development our industry found new ways of using images and containers. Images of our applications and services became deployment units for tools like Kubernetes, Docker Swarm or Marathon.
+When my OPS teammates chose a more conservative approach, it didn't change my joy of using Docker. Despite many people describing Docker as a tool written by developers for development, our industry found new ways of using images and containers. Images of our applications and services became deployment units for tools like Kubernetes, Docker Swarm or Marathon.
```

Comment:
It's your call :)
This would be correct grammar:

"While my ops teammates chose a more conservative approach, I started to use Docker with great joy."